azafea/tests/integration/__init__.py

```diff
@@ -42,7 +42,8 @@ def ensure_no_tables(self):
                 with self.db as dbsession:
                     dbsession.query(model).all()
 
-            assert f'relation "{model.__tablename__}" does not exist' in str(exc_info.value)
+            assert model.__tablename__ in str(exc_info.value)
+            assert 'UndefinedTable' in str(exc_info.value)
```

Comment:
I could be wrong but I think the exception will be an instance of this, which is a subclass of ProgrammingError – could we adjust the `with pytest.raises()` above to `with pytest.raises(UndefinedTable)`?

Reply:
I’m not sure. I think that it’s a psycopg2 error embedded in an SQLAlchemy error, not a subclass, so `raises` won’t work.

Reply:
It’s not an instance of `UndefinedTable`. I had already searched a long time ago for a way to get the original exception, I think that I didn’t find a clean solution. Did I miss something?

Reply:
No, I'm probably just wrong :-)

Reply:
(Unless it's a [chained exception](https://docs.python.org/3/reference/simple_stmts.html#raise) with `raise e1 from e2`?)

Reply:
> (Unless it's a [chained exception](https://docs.python.org/3/reference/simple_stmts.html#raise) with `raise e1 from e2`?)

It’s not. But thank you for the link, I didn’t know chained exceptions!